django/contrib/postgres/indexes.py

```diff
@@ -11,11 +11,11 @@ class BrinIndex(Index):
     # applicable.
     max_name_length = 31
 
-    def __init__(self, fields=[], name=None, pages_per_range=None):
+    def __init__(self, fields=[], name=None, pages_per_range=None, *, db_tablespace=None):
```

Comment:
I wonder if it would be better if index subclasses accepted `**kargs` instead, similar to 43b574007ef5edfe11f3b899fbe0a6fc05c43774. This could be done as a separate change that precedes this one. After de42adf4ff6b80e25bbd9e70070146ef15076a83, `GinIndex` would also need to be modified.